src/drivers/gps/gps.cpp

```diff
@@ -172,9 +173,10 @@ class GPS : public ModuleBase<GPS>
 	const Instance 			_instance;
 
 	uORB::Subscription		_orb_inject_data_sub{ORB_ID(gps_inject_data)};
-	orb_advert_t			_dump_communication_pub{nullptr};		///< if non-null, dump communication
+	uORB::PublicationQueued<gps_dump_s>	_dump_communication_pub{ORB_ID(gps_dump)};
 	gps_dump_s			*_dump_to_device{nullptr};
 	gps_dump_s			*_dump_from_device{nullptr};
+	bool				_dump_communication{false};			///< if true, dump communication
```

Comment:
I find the comment is not necessary. If it's not clear enough, what about calling it `_should_dump_communication`?

Reply:
Updated.